src/index.ts

```diff
@@ -1,7 +1,5 @@
 import { getAsyncLifecycle, defineConfigSchema } from "@openmrs/esm-framework";
 import { configSchema } from "./config-schema";
-import { registerExpressionHelper } from "@openmrs/openmrs-form-engine-lib";
```

Comment:
Please remove the form engine dependency from the `package.json` as well

Reply:
Thanks @pirupius will do so